ga4mp/ga4mp.py

```diff
@@ -20,18 +21,13 @@
 
 class Ga4mp(object):
```

Comment:
Would name BaseGA4MP to denote that it is a base class and not something you'd make an instance of to use the package.

Reply:
Include 
_build_url, _build_request

Something line:
```
def _build_url(self, domain):
    raise NotImplementedError()
```

Reply:
Updated the function names and the references to include the preceding underscore, then added these two functions to the base class:
```    
def _build_url(self, domain):
    raise NotImplementedError("Subclass should be using this function, but it was called through the base class instead.")

def _build_request(self, batch):
    raise NotImplementedError("Subclass should be using this function, but it was called through the base class instead.")```